src/data/showcases.js

```diff
@@ -1211,6 +1211,15 @@ export const Showcases = [
     source: null,
     tags: ["daotool", "governance"],
   },
+  {
+    title: "Nio",
+    description:
+      "Track all assets in wallets and locked in Cardano DeFi smart contracts. Effortlessly monitor your assets and whale wallets",
+    preview: require("./showcase/Nio_app.png"),
+    website: "https://nioapp.io",
+    source: null,
+    tags: ["analytics", "defi"],
```

Comment:
```suggestion
    tags: ["defi"],
```
I don't think the term `analytics` applies here, which you can see is generally applied toward applications that show a general view of the blockchain or insight into Ledger calculations: https://developers.cardano.org/showcase/?tags=analytics

Reply:
Thanks a lot for the feed back, I have updated the tags